omod/src/main/resources/config.xml

```diff
@@ -40,17 +44,22 @@
 	 /Required Privileges -->
 
 
-	<!-- Required Global Properties 
+	<!-- Required Global Properties-->
 	<globalProperty>
-		<property>@MODULE_ID@.someProperty</property>
-		<defaultValue></defaultValue>
+		<property>@MODULE_ID@.concept</property>
+		<defaultValue>11557d80-af43-4788-b30c-69bc60489814</defaultValue>
```

Comment:
It's preferable to support any mechanism for identifying a concept. UUIDs should be the last choice after mappings, etc. I'd recommend using `getConceptByReference()` to be able to support both.